dev.fredol.open-tv.yaml

```diff
@@ -0,0 +1,628 @@
+id: dev.fredol.open-tv
+
+runtime: org.gnome.Platform
+runtime-version: "46"
```

Comment:
you should use 47

Reply:
I tried to but I encountered a lot of build errors with mpv deps